docs/fhir-guide/FHIR 101 - Practical Guide.ipynb

```diff
@@ -0,0 +1,2966 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "hidden": true
+   },
+   "source": [
+    "### FHIR is NOT ...\n",
+    "- A database\n",
+    "- A database schema\n",
+    "- A server\n",
+    "\n",
+    "It is important to understand that the FHIR specification is completely technology agnostic. Thus, it does not depend on programming languages or include things like relational database schemas. It is up to the implementers to decide how to implement the data model (i.e. relational database, nosql database, etc) and RESTful API. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "heading_collapsed": true
+   },
+   "source": [
+    "## Concepts"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "hidden": true
+   },
+   "source": [
+    "The FHIR spec is large and daunting, so we cannot cover all of the major concepts in FHIR. However, this section and the next should give you all you need to understand what you're doing in the tutorial. \n",
+    "\n",
+    "📘A good resource to learn [FHIR Basics](https://smilecdr.com/docs/tutorial_and_tour/fhir_basics.html#fhir-basics)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "heading_collapsed": true,
+    "hidden": true
+   },
+   "source": [
+    "### FHIR Resource "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "hidden": true
+   },
+   "source": [
+    "The term \"Resource\" can be confusing as it is sometimes used to describe different concepts in FHIR. \n",
+    "\n",
+    "This diagram should help illustrate the difference between a Resource, type of Resource, and group/category of Resource types:\n",
+    "\n",
+    "![FHIR Concepts](static/images/resource-breakdown.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "hidden": true
+   },
+   "source": [
+    "#### Resource Groups, Types, and Instances\n",
+    "**Resource** - an instance of a type of thing. A Resource has an identifier. Example resources are:\n",
+    "- Patient `PT-001`\n",
+    "- Specimen `SP-001`\n",
+    "- StructureDefinition `KidsFirst-Patient`\n",
+    "- SearchParameter `patient-race`\n",
+    "- ...\n",
+    "\n",
+    "**Resource Type** - a type of a thing. Example resource types are:\n",
+    "- Patient\n",
+    "- Specimen\n",
+    "- StructureDefinition\n",
+    "- SearchParameter\n",
+    "- ...\n",
+    "\n",
+    "**Resource Type Category/Group** - a logical grouping of Resources. Examples are:\n",
+    "- [Conformance Resources](#Conformance-Resources)\n",
+    "- [Terminology Resources](#Terminology-Resources)\n",
+    "- ...\n",
+    "\n",
+    "The most important categories of resource types for the purposes of this tutorial are `Conformance Resources` and `Terminology Resources` because these are the ones we will create in order to build a FHIR data model. You will learn about them in the [Model Development](#Model-Development) section\n",
```

Comment:
```suggestion
    "The most important categories of resource types for the purposes of this tutorial are `Conformance Resources` and `Terminology Resources`, because these are the ones we will create in order to build a FHIR data model. You will learn about them in the [Model Development](#Model-Development) section\n",
```